tests/subgraph/minter-suite/minter-suite-mapping.test.ts

```diff
@@ -3174,3 +3178,168 @@ describe("MinterHolder-specific tests", () => {
+      ProjectMaxInvocationsLimitUpdated
+    >(newMockEvent());
+    event.address = minterAddress;
+    event.parameters = [
+      new ethereum.EventParam(
+        "_projectId",
+        ethereum.Value.fromUnsignedBigInt(projectId)
+      ),
+      new ethereum.EventParam(
+        "_maxInvocations",
+        ethereum.Value.fromUnsignedBigInt(BigInt.fromI32(100))
+      )
+    ];
+
+    event.block.timestamp = CURRENT_BLOCK_TIMESTAMP;
+
+    assert.notInStore(PROJECT_ENTITY_TYPE, fullProjectId);
+
+    handleProjectMaxInvocationsLimitUpdated(event);
+
+    assert.notInStore(PROJECT_MINTER_CONFIGURATION_ENTITY_TYPE, fullProjectId);
+  });
+
+  test("should do nothing if event maxInvocations are greater than core contract project maxInvocations", () => {
+    clearStore();
+    const minter = addNewMinterToStore("MinterSetPriceV3");
+    const minterAddress: Address = changetype<Address>(
+      Address.fromHexString(minter.id)
+    );
+    const minterType = minter.type;
+
+    const projectId = BigInt.fromI32(0);
+    const project = addNewProjectToStore(
+      TEST_CONTRACT_ADDRESS,
+      projectId,
+      "project 0",
+      randomAddressGenerator.generateRandomAddress(),
+      ONE_ETH_IN_WEI.div(BigInt.fromI32(10)),
+      CURRENT_BLOCK_TIMESTAMP.minus(BigInt.fromI32(10))
+    );
+
+    const projectMinterConfig = new ProjectMinterConfiguration(
+      getProjectMinterConfigId(minterAddress.toHexString(), project.id)
+    );
+    projectMinterConfig.minter = minterAddress.toHexString();
+    projectMinterConfig.project = project.id;
+    projectMinterConfig.extraMinterDetails = "{}";
+    projectMinterConfig.basePrice = ONE_ETH_IN_WEI.div(BigInt.fromI32(10));
+    projectMinterConfig.priceIsConfigured = false;
+    projectMinterConfig.currencyAddress = Address.zero();
+    projectMinterConfig.currencySymbol = "ETH";
+    projectMinterConfig.purchaseToDisabled = false;
+    projectMinterConfig.maxInvocations = null;
+    projectMinterConfig.save();
+
+    const event: ProjectMaxInvocationsLimitUpdated = changetype<
+      ProjectMaxInvocationsLimitUpdated
+    >(newMockEvent());
+    event.address = minterAddress;
+    event.parameters = [
+      new ethereum.EventParam(
+        "_projectId",
+        ethereum.Value.fromUnsignedBigInt(projectId)
+      ),
+      new ethereum.EventParam(
+        "_maxInvocations",
+        ethereum.Value.fromUnsignedBigInt(BigInt.fromI32(ONE_MILLION * 2))
+      )
+    ];
+
+    event.block.timestamp = CURRENT_BLOCK_TIMESTAMP;
+
+    handleProjectMaxInvocationsLimitUpdated(
+      changetype<ProjectMaxInvocationsLimitUpdated>(event)
+    );
+
+    const projectMinterConfigPostEvent = new ProjectMinterConfiguration(
+      getProjectMinterConfigId(minterAddress.toHexString(), project.id)
+    );
+    assert.fieldEquals(
+      PROJECT_MINTER_CONFIGURATION_ENTITY_TYPE,
+      getProjectMinterConfigId(minterAddress.toHexString(), project.id),
```

Comment:
Nit: should just save this in a variable so we don't need to regenerate in multiple places